healthScore/templates/record_submit_complete.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Request Sent</title>
+<style>
+  body {
+    font-family: Arial, sans-serif;
+    text-align: center;
+  }
+  header {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding: 20px;
+      background-color: #FFFFFF;
+  }
+  h2 {
+    text-align: center;
+  }
+  .logo {
+      width: 350px;
+      height: 100px;
+  }
+  .header_subtext {
+    text-align: center;
+  }
+  button {
+        background-color: #5B0888;
+        color: white;
+        padding: 14px 20px;
+        margin: 8px 0;
+        border: none;
+        border-radius: 4px;
+        cursor: pointer;
+  }
+  button:hover {
+      background-color: #360451;
+      color:white;
+  }
+  .link_text_modified {
+    text-decoration: none;
+    color: #FFFFFF;
+  }
+</style>
+</head>
+
+<header>
```

Comment:
Could you please import the header.html file that Sam had added? We can use that header to keep things uniform